WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2262,7 +2263,10 @@ PrefRowCheckWidget          = padding:2px 0;
 PrefRowSelectWidget         = min-width:280px; width:auto; max-width:100%;
 PrefRowInlineTable          = padding:0 @PrefViewInnerSpacing@;
 CheckWidgetInputText        = padding-top:0; padding-bottom:0; height:auto; line-height:1;
-
+PrefHeaderComment           = width:99%; margin-bottom: 24px;
```

Comment:
Any specific reason for `99%`? I know it was just moved from CSS to property, but can `@FullWidth@` work here?  
Just that it looks bit confusing to me and looks like some hack :)

Thanks